src/api.js

```diff
@@ -1,4 +1,30 @@
+// //Static Data for Front-End only testing & development
+// let people = [
+//   {
+//     first_name: "Jake",
+//     image_url: "https://d29xw0ra2h4o4u.cloudfront.net/assets/people/jake_davis_150-0d80259bc891893e234e2f2759cad60c483812f9dbfe1f3fa6a451e6415ccdc6.jpg",
+//     last_name: "Davis",
+//     middle_name: "",
+//     person_id: 1557
+//   }, {
+//     first_name: "Gonzalo",
+//     image_url: "https://d29xw0ra2h4o4u.cloudfront.net/assets/people/gonzalo_bulnes_guilpain_150-a6f7ee94cb1bd2dd4ab27a678eed3958086ffefa6acb1ac174d3038eaecf1436.jpg",
+//     last_name: "Bulnes Guilpain",
+//     middle_name: "",
+//     person_id: 1378
+//   },
+//   {
+//     first_name: "Patricia",
+//     image_url: "https://d29xw0ra2h4o4u.cloudfront.net/assets/people/patricia_estridge_150-1e7989ef0d09a7928ea46a199432f412904add6b4defd4855dd80316b8ce6eef.jpg",
+//     last_name: "Estridge",
+//     middle_name: "",
+//     person_id: 587
+//   }
```

Comment:
Adding placeholder data is a good idea! We probably shouldn't use real people, though. When I was initially developing, I used flower pictures from Wikipedia, but any kinda of non-personal data should be fine.